src/Sarif/Baseline/DefaultBaseline/LocationBaselineEquals.cs

```diff
@@ -13,12 +13,7 @@ public bool Equals(Location x, Location y)
         {
             if (!object.ReferenceEquals(x, y))
             {
-                // Target and Result file should match.
```

Comment:
Shouldn't we have moved the analysisTarget-related code into ResultBaselineEquals.cs?

**CAUTION** I don't know anything about the "Baseline" directory. I don't even know if the code here is used. Please look in the history, find the owner, and decide what's the right thing to do with this code.

Reply:
This is all new work from Everett that just went in last week. I'm adding a new equality class for FileLocation so that check can be added to ResultBaselineEquals.